### DISCUSSION TITLE
Lunch and learn

### DISCUSSION THREAD

**Discussion by katskillskat** [katskillskat/katskills-org#2] [Ideas]
I would love if people could get together to learn about topics like:

-Taxes
-Investing
-Mortgages
-Health insurance
-Birds and plants of the catskills
-Transportation options
-Pottery

Are there other people in the area who want to learn about these things? I'm sure that we could find local experts to come talk about a topic for 30-60mins in a local park. What things do other people want to learn?

**Comment by MVilleRobin** [comment]
I love this idea! I believe there are people who would be interested.

**Reply by katskillskat** [reply]
How do you think it should be setup? Is there a place where people can say: "here are some topics where I'm knowledgable" and then another person could schedule a time and place for the first person to talk about a topic for an hour? How would we get the word out.

I'm more than happy to help from a technology standpoint, like a calendar and signup system. Also know someone who would sponsor an ad in the local paper.

Meetup.com has some good tooling that we could embed in a website or direct people toward as well.
